Sources/NIOIMAPCore/Grammar/Response/UIDBatchesResponse.swift

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import struct NIO.ByteBuffer
+
+/// Sent from a server in response to an extended search.
+public struct UIDBatchesResponse: Hashable, Sendable {
+    /// The tag of the command that this search result is a response to.
+    public var correlator: String
```

Comment:
I think this should use `SearchCorrelator`? RFC 7377 updates `search-correlator` to accept multiple fields.

Reply:
Interesting.